src/response_decoder.ts

```diff
@@ -76,6 +153,12 @@ class BufferDataReader implements DataReader {
     return ret
   }
 
+  readToBuffer(size: number): Buffer {
```

Comment:
```suggestion
  readBuffer(size: number): Buffer {
```
Like readInt8, readInt16, readBuffer read a buffer